System.IO.Ports/SerialPort.cs

```diff
@@ -13,17 +13,19 @@ public sealed class SerialPort : IDisposable
     {
         // default new line
         [System.Diagnostics.DebuggerBrowsable(Diagnostics.DebuggerBrowsableState.Never)]
-        private const string _defaultNewLine = "\n";
+        private const string DefaultNewLine = "\n";
```

Comment:
This is not inline with .NET code style. Private fields start with _ and lower case. 
If this is throwing an error maybe it's our Stylecop rules that needs a tweak? 
cc @Ellerbach

Reply:
Reverted for the moment.